saleor/graphql/order/mutations/orders.py

```diff
@@ -424,29 +424,29 @@ class OrderUpdateMeta(UpdateMetaBaseMutation):
     class Meta:
         description = "Updates meta for order."
         model = models.Order
-        permissions = "orders.manage_orders"
```

Comment:
Maybe it is a good moment to create some permissions validator in `BaseMutation`? We just need some assert between expected permission and our list of available permission.
@maarcingebala What do you think?

Reply:
Not sure if I understand correctly, how is `BaseMutation` supposed to validate permissions? I think the issue is that we're relying on strings to identify permissions across the entire codebase, while we should have one enum for all permissions and use it instead.

Reply:
Yes, it would be a good time to change that now or very soon.

Reply:
> Not sure if I understand correctly, how is `BaseMutation` supposed to validate permissions? I think the issue is that we're relying on strings to identify permissions across the entire codebase, while we should have one enum for all permissions and use it instead.

it can be a simple assert - `meta_permission in MODELS_PERMISSIONS` 🤷‍♂ 
I agree that, the better way would be not using `strings`.